docs/core/versions/index.md

```diff
@@ -124,26 +128,29 @@ It's also possible that we need to fix the .NET Core Tools but reship an existin
 MAJOR.MINOR.PATCH[-PRERELEASE-BUILDNUMBER]
 ```
 
-The optional `PRERELEASE` and `BUILDNUMBER` parts will never be part of supported releases, and only exist on nightly builds, locally built from source targets, and unsupported preview releases.
+The optional `PRERELEASE` and `BUILDNUMBER` parts are never part of supported releases, and only exist on nightly builds, locally built from source targets, and unsupported preview releases.
 
 ### How version numbers are incremented?
 
 `MAJOR` is incremented when:
-  - An old version is no longer supported.
-  - A newer `MAJOR` version of an existing dependency is adopted.
-  - The default setting of a compatibility quirk is changed to "off."
+
+- An old version is no longer supported.
+- A newer `MAJOR` version of an existing dependency is adopted.
+- The default setting of a compatibility quirk is changed to "off."
 
 `MINOR` is incremented when:
-  - Public API surface area is added.
-  - A new behavior is added.
-  - A newer `MINOR` version of an existing dependency is adopted.
-  - A new dependency is introduced.
-  
+
+- Public API surface area is added.
+- A new behavior is added.
+- A newer `MINOR` version of an existing dependency is adopted.
+- A new dependency is introduced.
+
 `PATCH` is incremented when:
-  - Bug fixes are made.
-  - Support for a newer platform is added.
-  - A newer `PATCH` version of an existing dependency is adopted.
-  - Any other change that doesn't fit one of the previous cases.
+
+- Bug fixes are made.
+- Support for a newer platform is added.
+- A newer `PATCH` version of an existing dependency is adopted.
+- Any other change that doesn't fit one of the previous cases.
```

Comment:
"that" can be deleted here ("Any other change doesn't fit one of the previous cases.")

Reply:
is it wrong? otherwise it sounds better with it for me

Reply:
I'm fine with it either way, but since this is an expansion of "Patch is incremented when", it can be read as, "Patch is incremented when any other change that doesn't fit one of the previous cases." That interferes with the meaning; so "Path is incremented when any other change doesn't fit one of the previous cases." sounds better.